src/DocumentFormat.OpenXml/Packaging/OpenXmlPartExtensions.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using DocumentFormat.OpenXml.Features;
+using System;
+
+namespace DocumentFormat.OpenXml.Packaging
+{
+    /// <summary>
+    /// Defines extensions for part relationships
+    /// </summary>
+    public static class OpenXmlPartExtensions
+    {
+        internal static OpenXmlPart InitPart(this OpenXmlPartContainer parent, OpenXmlPart childPart, PartTypeInfo partType, string? relId = null)
+        {
+            if (parent == null)
+            {
+                throw new ArgumentNullException(message: "Parent part MUST be provided.", paramName: nameof(parent));
```

Comment:
```suggestion
                throw new ArgumentNullException(nameof(parent));
```

We generally don't provide a message for this as it's self explanatory.